README.md

```diff
@@ -0,0 +1,26 @@
+# PlayKit Google Cast iOS
+Google Cast support for [Kaltura Player](https://github.com/kaltura/kaltura-player-ios) and [PlayKit iOS](https://github.com/kaltura/playkit-ios)
+
+## Installation
+
+PlayKit Google Cast is available through [CocoaPods](http://cocoapods.org). To install
+it, simply add the following line to your Podfile:
+```ruby
+pod 'PlayKitGoogleCast'
+```
+
+## Example
+
+To run an PlayKit Google Cast smaple use:   
+https://github.com/kaltura/kaltura-player-ios-samples   
+https://github.com/kaltura/playkit-ios-samples/tree/develop/ChromecastSample    
+
+Additional documentation about Google Cast SDK:   
+https://developers.google.com/cast/docs/ios_sender  
+https://developers.google.com/cast/docs/release-notes
+
```

Comment:
@chausov  please add link to sample
please add explanation how to use it in 
OVP/OTT/Basic

how to use using IMA

how to manage live (start position -1) 
how to give start position
how to give tracks

see andorid: https://github.com/kaltura/playkit-android-googlecast